include/trsp/pgr_trspHandler.h

```diff
@@ -79,7 +79,8 @@ class Pgr_trspHandler {
          bool isIllegal(size_t i) {return v_pos[i] == ILLEGAL;}
          bool isIllegal(Position i) {
              pgassert(i != ILLEGAL);
-             return v_pos[i] == ILLEGAL;}
+             if (i == ILLEGAL) return true;
```

Comment:
That we can not, because:
pgassert with throw when that is not true but that only happens when the compilation is using `Debug` but when it is using  `Release` the next line will create a segmentation fault.